azure-iot-device/azure/iot/device/iothub/aio/async_clients.py

```diff
@@ -384,25 +384,31 @@ async def notify_blob_upload_status(
         await handle_result(callback)
         logger.info("Successfully notified blob upload status")
 
-    # @property
-    # def on_method_request_received(self):
-    #     return self._on_method_request_received
-
-    # @on_method_request_received.setter
-    # def on_method_request_received(self, value):
+    # def _generic_handler_setter(self, handler_name, feature_name, new_handler):
```

Comment:
># def _generic_handler_setter(self, handler_name, feature_name, new_handler): [](start = 4, length = 77)

This commented code will be refined and made available in the next PR - I left it in here because while it works, it's not something I wish to expose yet.